src/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation/build/netstandard2.0/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.targets

```diff
@@ -36,7 +36,7 @@
   <Target
     Name="_RunForCore"
     DependsOnTargets="_CreateResponseFileForMvcRazorPrecompile"
-    Condition="'$(TargetFrameworkIdentifier)'=='.NETCoreApp'">
+    Condition="'$(TargetFrameworkIdentifier)'=='.NETCoreApp' AND '$(RuntimeIdentifier)'==''">
```

Comment:
Is `RuntimeIdentifier` sufficient? We used to check that as well as `RuntimeIdentifiers`, https://github.com/aspnet/MetaPackages/blob/35ec06975d3b3738e83c6381d587c7c1d9e2eaf1/src/Microsoft.AspNetCore.All/build/PublishWithAspNetCoreTargetManifest.targets#L3